recipes/vsgimgui/all/conanfile.py

```diff
@@ -0,0 +1,130 @@
+            check_min_cppstd(self, self._min_cppstd)
+        check_min_vs(self, 191)
+        
+        if is_msvc_static_runtime(self):
+            raise ConanInvalidConfiguration(f"{self.name} does not support MSVC MT/MTd configurations, only MD/MDd is supported")
+            
+        if not is_msvc(self):
+            minimum_version = self._compilers_minimum_version.get(str(self.info.settings.compiler), False)
+            if minimum_version and Version(self.info.settings.compiler.version) < minimum_version:
+                raise ConanInvalidConfiguration(
+                    f"{self.ref} requires C++{self._min_cppstd}, which your compiler does not support."
+                )
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], destination=self.source_folder, strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+
+        if is_msvc(self):
+            tc.variables["USE_MSVC_RUNTIME_LIBRARY_DLL"] = not is_msvc_static_runtime(self)
+        
+        tc.variables["VSG_MACRO_PATH"] = os.path.join(self.deps_cpp_info["vsg"].rootpath, "lib/cmake/vsg/").replace("\\","/")
+    
+        tc.generate()
+        
+        deps = CMakeDeps(self)
+
+        deps.generate()
+
+    def imports(self):
+        self.copy("imgui_impl_vulkan.*", dst=os.path.join(self.source_folder,"src", "imgui", "backends"), src="./res/bindings")
+                  
+    def build(self):
+        apply_conandata_patches(self)
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, pattern="LICENSE.md", dst=os.path.join(self.package_folder, "licenses"), src=self.source_folder)
+        
+
+        cmake = CMake(self)
+        cmake.install()
+
+        rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
+        rmdir(self, os.path.join(self.package_folder, "share"))
+        rm(self, "*.la", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "lib"))
+        rm(self, "*.pdb", os.path.join(self.package_folder, "bin"))
+        rm(self, "Find*.cmake", os.path.join(self.package_folder, "lib/cmake/vsg"))
+        rm(self, "*Config.cmake", os.path.join(self.package_folder, "lib/cmake/vsg"))
+
+    def package_info(self):
+        self.cpp_info.libs = collect_libs(self)
+
+        self.cpp_info.set_property("cmake_file_name", "vsgImGui")
+        self.cpp_info.set_property("cmake_target_name", "vsgImGui::vsgImGui")
+
+        if self.settings.os in ["Linux", "FreeBSD"]:
+            self.cpp_info.system_libs.append("pthread")
+
+        # TODO: to remove in conan v2 once cmake_find_package_* generators removed
+        self.cpp_info.filenames["cmake_find_package"] = "vsgImGui"
+        self.cpp_info.filenames["cmake_find_package_multi"] = "vsgImGui"
+        self.cpp_info.names["cmake_find_package"] = "VSGIMGUI"
+        self.cpp_info.names["cmake_find_package_multi"] = "vsgImGui"
```

Comment:
```suggestion
```
Being this a new recipe, at the moment we are I think we can omit these and also remove the `test_v1_package`